tests/conftest.py

```diff
@@ -121,3 +96,20 @@ def disable_query_cache() -> Generator[None, None, None]:
     state.set_configs({"use_cache": 0, "use_readthrough_query_cache": 0})
     yield
     state.set_configs({"use_cache": cache, "use_readthrough_query_cache": readthrough})
+
+
+@pytest.fixture
+def set_state() -> Generator[Callable[[str, Any], None], None, None]:
```

Comment:
Is this needed? I don't see any place it's being referenced

Reply:
Yeah should have been removed.